FIX: prod_location (int) has no attribute 'id'

kholioeg opened the pull request:
`prod_location = location.id`  which returns type int.

This has generated an error mentioned and the fix is by removing the `id` attribute from `prod_location` part:

Line 85:  
`val['location_id'] = prod_location.id  `
To:
` val['location_id'] = prod_location`

kholioeg committed:
FIX: prod_location (int) has no attribute 'id' 

prod_location = location.id  which return type int. 

This has generated error mentioned and the fix is by removing the id attribute from prod_location part:

Line 85:  val['location_id'] = prod_location.id  
To:
 val['location_id'] = prod_location

pedrobaeza commented:
Please fix it the other way, removing the `.id` part in the variable assignment.

kholioeg committed:
Change the variable assignment in the top level.

Instead of removing the id attribute, assign prod_location variable with location object. 

prod_location = location

kholioeg reviewed (commented):
I've updated the code as requested
